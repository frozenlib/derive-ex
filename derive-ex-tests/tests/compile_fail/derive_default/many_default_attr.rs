use derive_ex::derive_ex;

#[derive_ex(Default)]
enum X {
    #[default]
    A,
    B,
    #[default]
    C,
}
fn main() {}
