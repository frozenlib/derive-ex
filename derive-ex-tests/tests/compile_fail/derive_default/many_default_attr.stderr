error: there are multiple variants with `#[default(...)]` (A, C)
 --> tests/compile_fail/derive_default/many_default_attr.rs:5:5
  |
5 |     #[default]
  |     ^
